README.md

```diff
@@ -1,18 +1,27 @@
-# twiliochat-android
+<a href="https://www.twilio.com">
+  <img src="https://static0.twilio.com/marketing/bundles/marketing/img/logos/wordmark-red.svg" alt="Twilio" width="250" />
+</a>
+
+# Twilio chat - Android
 [![Build Status](https://travis-ci.org/TwilioDevEd/twiliochat-android.svg?branch=master)](https://travis-ci.org/TwilioDevEd/twiliochat-android)
 
-Android implementation of Twilio Chat
+Learn to implement a simple chat application using Twilio Programmable Chat Client
```

Comment:
Would you mind adding a link to the [tutorial](https://www.twilio.com/docs/tutorials/walkthrough/chat/android/java) please?